php/unic/composer.json

```diff
@@ -1,26 +1,27 @@
 {
-  "name": "unicframework/unic",
-  "type": "project",
-  "description": "Unic is a high performance, open source web framework.",
-  "keywords": [
-    "unic",
-    "unic framework",
-    "php framework",
-    "web framework",
-    "mvt framework"
-  ],
-  "homepage": "https://unicframework.github.io/docs",
-  "license": "MIT",
-  "authors": [
-    {
-      "name": "Rajkumar Dusad",
-      "homepage": "https://github.com/rajkumardusad",
-      "role": "Developer"
-    }
-  ],
-  "require": {
-    "php": ">=5.6"
-  },
-  "minimum-stability": "dev",
-  "prefer-stable": true
+    "name": "unicframework/unic",
+    "type": "project",
+    "description": "Unic is a high performance, open source web framework.",
+    "keywords": [
+        "unic",
+        "unic framework",
+        "php framework",
+        "web framework",
+        "mvc framework"
+    ],
+    "homepage": "https://unicframework.github.io/docs",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Rajkumar Dusad",
+            "homepage": "https://github.com/rajkumardusad",
+            "role": "Developer"
+        }
+    ],
+    "require": {
+        "php": ">=5.6",
+        "unicframework/framework": "^1.0"
+    },
+    "minimum-stability": "dev",
+    "prefer-stable": true
```

Comment:
not sure it is required since it's the default behavior for **composer**

Reply:
Done